manifests/0000_51_olm_06_deployment.yaml

```diff
@@ -58,7 +58,7 @@ spec:
         - /cluster-olm-operator
         args:
         - start
-        - -v=2
+        - -v=4
```

Comment:
I _think_ this can be fully removed now that the logging controller is added. The verbosity can now be managed via the `OLM.spec.operatorLogLevel` field.